src/dms-preview/azext_dms/_help.py

```diff
@@ -5,6 +5,100 @@
 
 from knack.help_files import helps
 
+helps['dms task'] = """
+    type: group
+    short-summary: Manage service-level tasks for a Database Migration Service instance.
+    long-summary: |
+        Service-level tasks are non-migration tasks that are performed at the service level. This is usually for administrative and maintenance work which applies to the service's agent and VM.
+"""
+
+helps['dms task cancel'] = """
+    type: command
+    short-summary: Cancel a service-level Task if it's currently queued or running.
+"""
+
+helps['dms task create'] = """
+    type: command
+    short-summary: Create and start a service-level task.
+    long-summary: |
+        The following tasks are currently supported:
+            1) Check OCI driver against Oracle source to check for compatibility
+            2) Upload OCI driver install package
+            3) Install an already uploaded OCI driver package
+
+    parameters:
+        - name: --task-type
+          type: string
+          short-summary: >
+            The type of task to create. The supported types are CheckOciDriver, UploadOciDriver, and InstallOciDriver.
+        - name: --task-options-json
```

Comment:
json format is not a recommended in cli command because it is hard to validate and show allowed type. Is it possible to flatten the json object to several parameters?

Reply:
We had gone through our initial command review with Derek Bekoe (probably over a year ago at this point). Given the free-form nature of our inputs, it was decided back then that json was the best option. If you look at our base CLI as well as the extension here, json inputs are used throughout.

Reply:
json format is accepted in some scenarios but not recommended. If you can flatten it, it will be better to flatten json fromat to several parameters. In this way, we can give users more description for parameter usage and figure out which parameter is not correct.
But in some very complicated scenarios, we also ca accept json file.